ci_support/run_docker_build.sh

```diff
@@ -29,7 +29,7 @@ cat << EOF | docker run -i \
                         -v ${RECIPE_ROOT}:/recipe_root \
                         -v ${FEEDSTOCK_ROOT}:/feedstock_root \
                         -a stdin -a stdout -a stderr \
-                        condaforge/linux-anvil \
+                        pelson/obvious-ci:latest_x64 \
```

Comment:
This seems to revert some changes from the latest `rerender`. Can you try updating your fork?